TetrisEngine.java

```diff
@@ -0,0 +1,134 @@
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+
+public class TetrisEngine {
+	private static final String INPUT_FILE = "input.txt";
+	private static final String OUTPUT_FILE = "output.txt";
+	//private static final int GRID_SIZE = 100;
+	
+	private static HashMap<Character, TetrisPiece> PieceMap = new HashMap<Character, TetrisPiece>();

```

Comment:
These: (PieceMap, Grid) both look like instance variables to me, they should not be static. You should have a single java class with your main method whose only responsibility is to create a new instance of TetrisEngine. TetrisEngine should be a fully instance-capable class. This helps a ton with code reuse.